mathext/ell_complete.go

```diff
@@ -8,13 +8,17 @@ import (
 	"math"
 )
 
-// CompleteK computes the complete elliptic integral of the 1st kind, 0≤m≤1.
-// It returns math.NaN() if m is not in [0,1].
+// CompleteK computes the complete elliptic integral of the 1st kind, 0≤m≤1. It returns math.NaN() if m is not in [0,1].
 //
-//	K(m) = \int_{0}^{\pi/2} 1 / {\sqrt{1-m{\sin^2\theta}}} d\theta
-//
-// See: http://dx.doi.org/10.1016/j.cam.2014.12.038 for the computation method.
+//  K(m) = \int_{0}^{π/2} 1/{\sqrt{1-m{\sin^2θ}}} dθ
 func CompleteK(m float64) float64 {
+	// Reference:
+	// Toshio Fukushima, Precise and fast computation of complete elliptic integrals
+	// by piecewise minimax rational function approximation,
+	// Journal of Computational and Applied Mathematics, Volume 282, 2015, Pages 71-76.
+	// https://doi.org/10.1016/j.cam.2014.12.038
+	// Original Fortran code available at:
+	// https://www.researchgate.net/publication/295857819_xceitxt_F90_package_of_complete_elliptic_integral_computation
```

Comment:
Am I missing something? The code below looks wrong. It's using `t`, but the fortran code uses `t2`, i.e. `t*t`

Reply:
The same looks true also for `CompleteE`